packages/analytics/analytics-chart/src/components/SimpleChart.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="analytics-chart-shell">
+    <KEmptyState
+      v-if="!hasValidChartData"
+      class="chart-empty-state"
+      :cta-is-hidden="true"
+      data-testid="no-data-in-report"
+      icon="stateNoData"
+      icon-size="170"
+    >
+      <template #title>
+        {{ emptyStateTitle }}
+      </template>
+      <template #message>
+        {{ emptyStateDescription }}
+      </template>
+    </KEmptyState>
+    <div
+      v-else
+      class="analytics-chart-parent"
+    >
+      <DoughnutChart
+        v-if="isGaugeChart"
+        :chart-data="computedChartData"
+        :dataset-colors="chartOptions.chartDatasetColors"
+        is-simple
+        :legend-position="legendPosition"
+        :legend-values="legendValues"
+        :metric-display="chartOptions.metricDisplay"
+        :metric-unit="computedMetricUnit"
+        :synthetics-data-key="syntheticsDataKey"
+        :width="width"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import composables from '../composables'
+import { AnalyticsChartOptions } from '../types'
+import { ChartTypesSimple, ChartLegendPosition } from '../enums'
+import DoughnutChart from './chart-types/DoughnutChart.vue'
+import { computed, PropType, provide, toRef } from 'vue'
+import { AnalyticsExploreResult, AnalyticsExploreV2Result } from '@kong-ui-public/analytics-utilities'
+import { datavisPalette } from '../utils'
+
+const props = defineProps({
+  chartData: {
+    type: Object as PropType<AnalyticsExploreResult | AnalyticsExploreV2Result>,
+    required: true,
+  },
+  chartOptions: {
+    type: Object as PropType<AnalyticsChartOptions>,
```

Comment:
I would make this `PropType<SimpleChartOptions` after you address the comment I had on the chart options